packages/hiro-graph-client/index.d.ts

```diff
@@ -425,11 +432,17 @@ interface DefineVariableOptions {
   [key: string]: any;
 }
 
+interface SuggestVariable {
```

Comment:
Naming should be fixed. Maybe SuggestOptions?

Reply:
renamed to SuggestVariableOptions like DefineVariableOptions